packages/graphql-typescript-definitions/test/fixtures/all-clear/.graphqlconfig

```diff
@@ -2,6 +2,6 @@
   "schemaPath": "schema.json",
   "includes": ["documents/**/*.graphql"],
   "extensions": {
-    "schemaTypesPath": "schema.ts"
+    "schemaTypesPath": "types"
```

Comment:
i'm betting you can get rid of these extensions in the fixtures, they were put in place to force the output filename to be `schema.ts` but now that it's just a directory name it is no longer necessary.